src/components/model.js

```diff
@@ -90,4 +99,7 @@ module.exports.Component = registerComponent('loader', {
       el.setObject3D('mesh', collada.scene);
     });
   }
-});
+};
+
+module.exports.Component = registerComponent('loader', ModelComponent);
+module.exports.Component = registerComponent('model', ModelComponent);
```

Comment:
hm? most examples out there use models. we will remember to remove it, it's in the changelog.